docs/usage.md

```diff
@@ -0,0 +1,125 @@
+# Usage
```

Comment:
I would call this section something more specific, since we already have a "Usage" section in the main README. Maybe "Usage in hermetic builds" or "Hermetic build example"

Reply:
It really is supposed the general Usage doc though. It's the only one that describes the individual CLI commands and their options. I went with a hermetic build example because it was hard to explain the usage without one

Reply:
So maybe call the section in the main README "Basic Usage"? :sweat_smile:

Reply:
That works :smile: done